### PR TITLE
Updated to include -ForwardingAddress parameter email-forwarding.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1497

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by mattpennathe3rd** [comment]
@dariomws Are you able to add a few more details to this one? The referenced issue has a lot of useful info, so it might be helpful to incorporate some of that into this PR to better explain what the parameter does, or we can link to another article if it's already explained elsewhere.

If you're not sure or are not comfortable doing this, that is fine and I can have a SME take a closer look.

**Comment by dariomws** [comment]
> @dariomws Are you able to add a few more details to this one? The referenced issue has a lot of useful info, so it might be helpful to incorporate some of that into this PR to better explain what the parameter does, or we can link to another article if it's already explained elsewhere.
> 
> If you're not sure or are not comfortable doing this, that is fine and I can have a SME take a closer look.

Hi @mattpennathe3rd, sure I can add more information if you want. The information that the issue author provide us is from the parameter description on [Set-Mailbox](https://docs.microsoft.com/powershell/module/exchange/mailboxes/set-mailbox). And the article mention it: "For detailed syntax and parameter information, see Set-Mailbox."

Do you still think that we need to add more information? Maybe some information from [Set-Mailbox](https://docs.microsoft.com/powershell/module/exchange/mailboxes/set-mailbox) to this article.

Thanks!

**Comment by mattpennathe3rd** [comment]
@dariomws Sure, I asked because the examples in the article use two different parameters as a result of the PR, but there is no mention of why one is used in one cases versus the other. E.g., if a recipient is within the same organization versus outside (or whatever other considerations are in play).

**Comment by dariomws** [comment]
@mattpennathe3rd the description of the examples are as follow:

This example delivers email to the mailbox of Douglas Kohn and, at the same time, forwards all mail sent to Douglas Kohn to **an external email addresses,** douglaskohn.parents@fineartschool.net.

This example forwards all email sent to the mailbox of Ken Sanchez, an employee of Contoso Suites, to one of his coworkers, pilarp@contoso.com.

In bold you can see that I want to add. Are you agree? Or, Are you thinking in a longer description?

Thanks!

File: Exchange/ExchangeServer/recipients/user-mailboxes/email-forwarding.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ You need to be assigned permissions before you can perform this procedure or pro
 
 Haven't used Exchange Management Shell much? Check out the [Exchange Management Shell](https://docs.microsoft.com/powershell/exchange/exchange-server/exchange-management-shell) topic to learn more. Take a look at the [Get-Mailbox](https://docs.microsoft.com/powershell/module/exchange/mailboxes/get-mailbox) and [Set-Mailbox](https://docs.microsoft.com/powershell/module/exchange/mailboxes/set-mailbox) topics for more details on the cmdlets used here.
 
-This example delivers email to the mailbox of Douglas Kohn and, at the same time, forwards all mail sent to Douglas Kohn to douglaskohn.parents@fineartschool.net.
+This example delivers email to the mailbox of Douglas Kohn and, at the same time, forwards all mail sent to Douglas Kohn to a external email addresses, douglaskohn.parents@fineartschool.net.
```
Comment (mattpennathe3rd) [inline Exchange/ExchangeServer/recipients/user-mailboxes/email-forwarding.md:52]:
```suggestion
This example delivers email to the mailbox of Douglas Kohn and also forwards all mail sent to Douglas Kohn to an external email address, douglaskohn.parents@fineartschool.net.
```

**Comment by mattpennathe3rd** [comment]
@dariomws This works fine. Thanks for adding this and I'll merge this one now.

**Pull request merged**